components/Dialogs/examples/DialogsTypicalUseExampleViewController.m

```diff
@@ -25,24 +25,28 @@
 
 @interface DialogsTypicalUseExampleViewController : UIViewController
 @property(nonatomic, strong, nullable) MDCContainerScheme *containerScheme;
-@property(nonatomic, strong, nullable) MDCSemanticColorScheme *colorScheme;
 @property(nonatomic, strong, nullable) NSArray *modes;
 @property(nonatomic, strong, nullable) MDCButton *button;
 @end
 
 @implementation DialogsTypicalUseExampleViewController
 
+- (instancetype)init {
+  self = [super init];
+  if (self) {
```

Comment:
Tip: It makes more sense to me to conditionally (if not already) initialize the containerScheme in viewDidLoad instead of adding an initializer.  It's also more consistent with some other examples.

Reply:
If we did that we would have to conditionally check if `containerScheme` is nil then again check if that `containerScheme.colorScheme` is nil.